constraint/pkg/client/client.go

```diff
@@ -358,13 +358,18 @@ func (c *Client) AddConstraint(ctx context.Context, constraint *unstructured.Uns
 		return resp, clienterrors.ErrNoDriver
 	}
 
-	changed, err := cached.AddConstraint(constraint)
+	constraintWDefaults, err := c.withDefaultParams(constraint, template)
```

Comment:
let's just call it `constraintWithDefaults`, the "W" is more confusing than illuminating otherwise